lib/services/service_brokers/service_manager.rb

```diff
@@ -58,16 +58,34 @@ def update_or_create_plans(catalog)
           plan.public = false
         end
 
+        create_instance = nil
+        begin
+          create_instance = catalog_plan.schemas.service_instance.create.schema.try(:to_json)
```

Comment:
`create_instance = catalog_plan.schemas.service_instance.create.schema.try(:to_json) rescue nil` 

is more idiomatic and then you don't need the other 4 lines